Climate_police/tests/hw7-mao/pre_process.py

```diff
@@ -0,0 +1,27 @@
+from us_state_abbrev import us_state_abbrev
+
+def pre_process(df, source, year):
+
+	for col in df.columns:
+		df[col] = df[col].astype(str)
+    
+    # Add a column containing the text displayed when mouse hovering
+	df['text'] = df['state'] + '<br>' +\
+    	source + ' Mean '+df[source+' Mean']+'<br>'+\
```

Comment:
You are considering only the average values of the pollutants, but valuable information could be provided by the first maximum value registered or the units of the pollutants per each day!!!